website/content/v1.2/kubernetes-guides/configuration/replicated-local-storage-with-openebs-jiva.md

```diff
@@ -136,7 +136,7 @@ kubectl --namespace openebs patch daemonset openebs-jiva-csi-node --type=json --
 
 ## Testing a simple workload
 
-Now proceed on to testing the jiva installation, let's first create a PVC referencing the `openebs-jiva-csi-default` storage class:
+In order to test the Jiva installation, let's first create a PVC referencing the `openebs-jiva-csi-default` storage class:
 
 ```yaml
```

Comment:
feels like we should also detail what to expect from the test here. not clear what running the fio tests actually do.